cdap-ui/app/cdap/components/AbstractWidget/WranglerEditor/index.tsx

```diff
@@ -1,3 +1,16 @@
+import * as React from 'react';
```

Comment:
It looks a little weird to have some of the imports before the license.

Reply:
Whoops, will fix this!

Reply:
Resolved